app/src/logging/__init__.py

```diff
@@ -36,11 +37,16 @@
 _original_argv = tuple(sys.argv)
 
 
-def init(program_name: str) -> logging.Logger:
-    root_logger = config.configure_logging()
+class Log:
+    def __init__(self, program_name: str) -> None:
+        self.stream_handler = config.configure_logging()
+        log_program_info(program_name)
 
-    log_program_info(program_name)
-    return root_logger
+    def __enter__(self) -> None:
+        return None
+
+    def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
+        logging.root.removeHandler(self.stream_handler)
```

Comment:
could we consolidate this class into `config.py` and consolidate the configure_logging function into a method on this class so that `logging.root.addHandler` and `logging.root.removeHandler` are at the same level (i.e. in the same class)